server/pkg/item/field_change.go

```diff
@@ -0,0 +1,85 @@
+package item
+
+
+type FieldChangeType string
+
+const (
+	Add    FieldChangeType = "add"
+	Update FieldChangeType = "update"
+	Delete FieldChangeType = "delete"
+)
+
+
+type FieldChange struct {
+	Id *FieldID
+	CurrentValue any
+	PreviousValue any
+	Type FieldChangeType
+}
+
+
+func NewItemChange(n []*Field, o []*Field) []FieldChange {
```

Comment:
```suggestion
func CompareFields(n []*Field, o []*Field) []FieldChange {
```

Also add unit tests for this func.